.github/pull_request_template.md

```diff
@@ -1,9 +1,10 @@
-## Why was this change made? 🤔
```

Comment:
Get out of the practice of choosing a heading level based on how it looks vs. its semantics (an accessibility concern itself).

Reply:
i really wish there was a  way to downsize the headings across the board in a doc/comment/etc if desired, so that we could have less giant headings that still made sense semantically.  but alas, i couldn't find anything in the github markdown docs indicating that's possible.  but agree that this change is the right thing to do.

Reply:
Something something user-specific stylesheets, but I have no idea what that means in practice.